load-test-framework/src/main/java/com/google/pubsub/flic/Driver.java

```diff
@@ -141,6 +167,11 @@
           "message loss. If set less than 1, this flag is ignored."
   )
   private int numberOfMessages = 0;
+  
+  private final String SPREADSHEET_ID = System.getenv("LOADTEST_SPREADSHEET_ID");
```

Comment:
Make the default empty, this should be something the user enables. You can use an alias so that the flag is provided by default for actually running it.